.github/workflows/container_build.sh

```diff
@@ -4,7 +4,7 @@ yum install -y wget git gcc
 wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
 chmod a+x Miniconda3-latest-Linux-x86_64.sh
 ./Miniconda3-latest-Linux-x86_64.sh -b
-${HOME}/miniconda3/bin/conda create -y -n hexrd python=3.8
+${HOME}/miniconda3/bin/conda create -y -n hexrd python=3.9
```

Comment:
I'm wondering if we should add `--override-channels -c conda-forge` here to ensure we are using python from conda forge. However, that doesn't need to be done in this PR.

Reply:
I can add that in to be safe.